kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -2826,6 +2826,32 @@ object KyuubiConf {
       .version("1.7.2")
       .fallbackConf(ENGINE_SUBMIT_TIMEOUT)
 
+  object YARNClientProxyUserStrategy extends Enumeration {
+    type YARNClientProxyUser = Value
+    val NONE, ADMIN, OWNER = Value
+  }
+
+  val YARN_PROXY_USER_STRATEGY: ConfigEntry[String] =
+    buildConf("kyuubi.yarn.proxyUser.strategy")
```

Comment:
How about `kyuubi.yarn.client.proxyUser.xxx` ?

Reply:
I'm open to the configuration name, but there are existing `kyuubi.kubernetes.xxx` without `client` namespace too.